CMakeLists.txt

```diff
@@ -229,7 +229,7 @@ if(CMAKE_COMPILER_IS_GNUCC)
 	endif()
 endif()
 
-if("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang")
+if(${CMAKE_C_COMPILER_ID} STREQUAL "Clang")
```

Comment:
Side note: This is because of http://www.cmake.org/cmake/help/v3.1/policy/CMP0054.html